server/channels/store/localcachelayer/user_layer.go

```diff
@@ -85,6 +97,30 @@ func (s *LocalCacheUserStore) InvalidateProfilesInChannelCache(channelID string)
 	}
 }
 
+func (s *LocalCacheUserStore) GetAllProfiles(options *model.UserGetOptions) ([]*model.User, error) {
+	if isEmptyOptions(options) &&
+		options.Page == 0 && options.PerPage == 100 { // This is hardcoded to the webapp call.
+		// read from cache
```

Comment:
Good comment, something to keep in mind and maybe share with front-end teams to avoid unexpected regressions.

Reply:
Actually, let me add a comment in the webapp code. It's good that we now have the monorepo, so I can do everything in the same PR!

Reply:
Great idea!